scripts/release/update-sdk-ts-defs.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @fileoverview
+ * Updates the SDK Typescript definitions located in the file sdk.d.ts
+ * to match the current definitions of functions of sdk.mjs.
+ */
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { execSync } from 'node:child_process';
+import { getDirnameFromImportMeta } from '../../sdk.mjs';
+
+const __dirname = getDirnameFromImportMeta(import.meta.url);
+const rootDir = path.resolve(__dirname, '..', '..');
+
+const sdkTs = path.resolve(rootDir, 'sdk.d.ts');
+const sdkMts = path.resolve(rootDir, 'sdk.d.mts');
+const sdkMjs = path.resolve(rootDir, 'sdk.mjs');
+
+const generateSdkMts = () => {
+  // remove temporally type definitions imported with comments
+  // in sdk.mjs to avoid circular imports
+  const originalSdkMjsContent = fs.readFileSync(sdkMjs, 'utf-8');
+  const tempSdkMjsContent = originalSdkMjsContent
+    .split('\n')
+    .filter((line) => {
+      return !line.startsWith(' * @typedef {import("./sdk")');
+    })
+    .join('\n');
+  fs.writeFileSync(sdkMjs, tempSdkMjsContent);
+  try {
+    execSync(
+      'tsc sdk.mjs --declaration --emitDeclarationOnly --allowJs --removeComments',
```

Comment:
```suggestion
      'npx tsc sdk.mjs --declaration --emitDeclarationOnly --allowJs --removeComments',
```

The `typescript` didn't install globally. We should use `npx` to run `tsx`.